library/src/main/java/com/google/maps/android/clustering/ClusterManager.java

```diff
@@ -193,7 +217,22 @@ public void removeItem(T item) {
     }
 
     /**
-     * Force a re-cluster. You may want to call this after adding new item(s).
+     * Updates an item in clusters. After calling this method you must invoke cluster() for the state
+     * of the clusters to be updated on the map.
+     * @param item item to update in clusters
+     */
+    public void updateItem(T item) {
```

Comment:
Thoughts on returning a bool here? It could be helpful to know if an item you are trying to update was actually updated or not (and if not, can signal to the developer that they need to implement equals and hashcode).

Reply:
:+1: That's a really good idea, and actually makes this testable too.  I went ahead and added returning boolean for all Algorithm add/remote/update operations to match Java Collection behavior in https://github.com/googlemaps/android-maps-utils/pull/627/commits/a1b1c17eac8d71ecbb0758bac6f1f514547c21f1, and added unit tests for them too. I could use another review on that just to make sure I didn't miss anything.